client/src/components/users/types.ts

```diff
@@ -39,12 +39,16 @@ export class UserCollection {
     return this.users.length;
   }
 
+  activeUsersCount(): number {
```

Comment:
You don't need to update these methods, it's better to treat a revoked user as not selectable, which means updated the `selectAll` method instead.

You can also delete the `revokedSelectedCount()` method.

Reply:
Done